internal/api/flags.go

```diff
@@ -31,3 +38,10 @@ func defaultFlags() *Flags {
 		insecureSkipVerify: &d,
 	}
 }
+
+func StreamingFlags(flagSet *flag.FlagSet) *Flags {
+	return &Flags{
+		trace:              flagSet.Bool("trace", false, "Log the trace ID for requests. See https://docs.sourcegraph.com/admin/observability/tracing"),
+		insecureSkipVerify: flagSet.Bool("insecure-skip-verify", false, "Skip validation of TLS certificates against trusted chains"),
+	}
```

Comment:
Why the duplication here? The `NewFlags` already returns a `*Flags` with both of these fields.

Edit: And wouldn't parsing fail if someone sets the `get-curl` flag, for example?

Reply:
> Edit: And wouldn't parsing fail if someone sets the get-curl flag, for example?
 
parsing only fails if someone calls `search` with `-stream` AND `-get-curl`, which is intended.  `-curl` and `-dump` make a lot less sense for streaming than for the GraphQL API. I assume the use case for `get-curl` is to pipe the output to something like `jq`? The streaming endpoint does not return valid JSON, but a stream of events that contain JSON.

Reply:
we could make get curl work though? My original streaming client was curl :D But yeah, until we have stabalized the API properly I don't want people to start writing there own clients.